.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         version: latest
     # Temporarily ignore the time/chrono segfault since there are no known
     # good wordarounds and they are used by dependent libraries also.
-    - run: cargo audit --ignore RUSTSEC-2020-0159 --ignore RUSTSEC-2020-0071
+    - run: cargo audit --ignore RUSTSEC-2020-0159 --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2023-0052
```

Comment:
Have we checked the previous two? Are they fixed.

Reply:
#403 , RUSTSEC-2020-0071 can be resolved by upgrading the versions of rust-s3 and some other libraries.

Reply:
RUSTSEC-2020-0159 is outdated.